feat: docs for conversation custom attributes via SDK (#397)
chatwoot/docs: docs/product/channels/live-chat/sdk/setup.md

window.$chatwoot.deleteCustomAttribute("attribute-key");
```

#### For a conversation

> You can also set custom attributes for the active conversation from the SDK. To set the custom attributes call `setConversationCustomAttributes` as follows

Reviewer: ```suggestion
You can also set custom attributes for the active conversation from the SDK. To set the custom attributes, call `setConversationCustomAttributes` as follows.
```

```js
window.$chatwoot.setConversationCustomAttributes({
  productName: "iPhone",
  productCategory: "Smartphone",
});
```

> You can view these information in the sidepanel of a conversation.

Reviewer: ```suggestion
You can view this information in the side panel of a conversation.
```

> To delete a custom attribute, use `deleteConversationCustomAttribute` as follows

Reviewer: ```suggestion
To delete a custom attribute, use `deleteConversationCustomAttribute` as follows.
```
